Sources/PackageCollections/PackageCollections+Configuration.swift

```diff
@@ -0,0 +1,19 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2020 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+// TODO: how do we read default config values? ENV variables? user settings?
```

Comment:
I think we should read these from a config file in ~/.swiftpm

Reply:
interesting. so @neonichu @abertelrud do you think that we should transition all of SwiftPM to a model in which we read config from `~/.swiftpm` and this will be the first step? if so, should we have one monolithic file or several per "domain"?

Reply:
My understanding is that `~/.swiftpm` was introduced relatively recently to reign in the various places where configuration could be stored, and that existing places should be transitioned as well.  I think the idea was to have a directory of separate files for different domains, to keep each one simple.  Boris might know the history here better than I.

Reply:
okay, I will take that approach then as we start adding options here